package.json

```
@@ -45,6 +45,7 @@
     "build": "npm run clean && npm run build:tsc && npm run build:webpack && npm run build:publish",
     "prepare": "npm run build",
     "lint": "eslint -c .eslintrc.js --ext .ts src",
+    "lint:fix": "eslint -c .eslintrc.js --ext .ts src --fix",
```

To reduce duplication, couldn't it be...
```suggestion
    "lint:fix": "npm run lint -- --fix",
```